soroban-sdk/src/testutils.rs

```diff
@@ -29,6 +31,83 @@ pub trait Ledger {
         F: FnMut(&mut LedgerInfo);
 }
 
+pub mod budget {
+    use core::fmt::Display;
+
+    #[doc(inline)]
+    pub use crate::env::internal::budget::CostType;
+
+    /// Budget that tracks the resources consumed for the environment.
+    ///
+    /// Note that some budgets – CPU instructions, memory, and VM cost types –
```

Comment:
This comment can be more clarified. The budget consist of two dimensions – CPU instructions, memory bytes. Each dimension consist of cost types, the WASM/VM-related cost types are the ones being omitted(underestimated).

Reply:
Thanks I updated the docs.